be/tool/beck.c

```diff
@@ -213,6 +215,8 @@ struct action_ops {
 };
 
 enum { CACHE_SIZE = 1000000 };
+enum { NV_OFFSET_SAVE_DELTA_IN_BYTES = 0x40000000 }; /* 1G */
+enum { NV_OFFSET_SAVE_ACT_DELTA = 1000 };
```

Comment:
Can be declared on one enum

Reply:
units are different. first one is in terms of bytes and second one is in terms of action count.
In case of single enum we can't use configuration which has same numerical value for both.